docker/deploy.sh

```diff
@@ -22,16 +23,29 @@ fi
 # Supply the 32 byte encryption key for AES256 as an environment variable
 #
 export ENCRYPTION_KEY=$(openssl rand 32 | xxd -p -c 64)
-echo -n $ENCRYPTION_KEY > encryption.key
+echo -n $ENCRYPTION_KEY > docker/encryption.key
 
 #
 # For Kong we must update a template file
 #
 if [ "$PROFILE" == 'kong' ]; then
-  envsubst < kong/kong.template.yml > ./kong/kong.yml
+  envsubst < docker/kong/kong.template.yml > docker/kong/kong.yml
 fi
 
+#
+# Build a custom Docker image, which uses 'luarocks make' to deploy the plugin
+#
+if [ "$PROFILE" == 'kong' ]; then
+  docker build -f docker/kong/Dockerfile --no-cache -t custom_kong:2.6.0-alpine .
```

Comment:
Now that we've confirmed that docker properly invalidates the cache when copied files change I think that `--no-cache` is not necessary.